Dockerfile

```diff
@@ -19,5 +19,5 @@ RUN gem update bundler
 # location where bundler is installed
 RUN GEM_HOME=/usr/local/lib/ruby/gems/2.2.0 gem cleanup bundler
 
-# install base version of rails
-RUN gem install rails -v 3.2.22.9 --source "https://concord:$RAILS_LTS_PASS@gems.railslts.com"
+# Configure bundler with rails lts credentials
+RUN bundle config gems.railslts.com concord:$RAILS_LTS_PASS
```

Comment:
It seems this isn't actually installing the gem.
One reason for this base image was to speed up the building and deploying of rigse images.
Perhaps it wasn't actually meeting that goal.
And perhaps the rails gem install was saving us much time.
Do you have comments on this?

Reply:
Ah, I didn't realize that the original code (Pre Nate's PR) actually installed Rails here.

In the case of `rigse`, the version of Rails installed here was never used by development because `Dockerfile-dev` sets `BUNDLE_PATH` to a volume (`/bundle`). However, I do believe rails installed here would benefit the production build as you have described.

So, I think the solution is to do both things. We can install Rails LTS here for the sake of production builds, but also configure bundler with credentials so that that the docker dev env can install the gem into the volume upon `bundle install`.

Does this sound good to you?

Reply:
Yup that sounds good.